sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -3044,6 +3059,60 @@ object Radar {
         }
     }
 
+    /**
+     * Flushes replays to the server.
+     */
+    @JvmStatic
+    internal fun flushReplays(currentTrackParams: JSONObject? = null, callback: RadarFlushReplaysCallback? = null) {
+        if (!initialized) {
+            return
+        }
+
+        // check if already flushing
+        if (flushingReplays) {
+            this.logger.i("Radar.flushReplays() already flushing", RadarLogType.SDK_CALL)
+            return // track callback?
+        }
+
+        // check if replays to flush
+        // get a copy of the replays so we can safely clear what was synced up
+        // if we have a current track update, add it to the local replay list
+        val replays = Radar.getReplays().toMutableList()
+        if (currentTrackParams != null) {
```

Comment:
You'll want to set aside a timestamp param now, prior to the request, in case the request takes a long amount of time